app/javascript/angular/code/controllers/note_ctrl.js

```diff
@@ -1,5 +1,5 @@
-function NoteCtrl($scope, note, singleMobileSession) {
-  var session = singleMobileSession.get();
+function NoteCtrl($scope, note, mobileSessions, sessionsUtils) {
+  var session = sessionsUtils.selectedSession(mobileSessions);
```

Comment:
I guess this is getting `mobileSessions` cause we have notes only for mobile sessions, correct?

Reply:
Yes, our mobile app expert confirmed we don't have notes for fixed sessions.